src/anaconda/install.sh

```diff
@@ -103,4 +103,22 @@ if ! conda --version &> /dev/null ; then
     updaterc "export CONDA_DIR=${CONDA_DIR}/bin"
 fi
 
+# Display a notice on conda when not running in GitHub Codespaces
+cat << 'EOF' > /usr/local/etc/vscode-dev-containers/conda-notice.txt
+When using "conda" from outside of GitHub Codespaces, note the Anaconda repository contains
+restrictions on commercial use that may impact certain organizations. See https://aka.ms/ghcs-conda
```

Comment:
FYI, we should repoint `https://aka.ms/ghcs-conda` to `https://github.com/devcontainers/features/blob/main/src/conda/NOTES.md#using-conda` instead of `https://github.com/microsoft/vscode-dev-containers/tree/main/containers/codespaces-linux#using-conda`

Reply:
@bamurtaugh Could you help change the target URL? I can see you listed as one of the link owners 😄

Reply:
Done! ✅